### DISCUSSION TITLE
WebSocket 基本介紹及使用筆記

### DISCUSSION THREAD

**Discussion by minniewei** [1101-GS4719-C/Python-Community-GS4719-C#515] [General]
最近老師在上http相關的課程內容的時候提到了websocker。我認為websocker是個很有趣的東西，透過websocker協定，server竟然可以主動和client端產生連接，有效的提升效率。未來當我有足夠的實力的時候，我希望能靠自己的實力用websocker 獨自產生一個網站。現在先跟大家節紹這個酷東東！！！！
https://www.letswrite.tw/websocket/

**Comment by hapiwangy** [comment]
之前在上課的時候曾經用過socket的技術來連接感應器相關的東西。但經過這篇文章的分享後了解到了如何在網站運用相關的技術。:）

**Comment by sean1325418** [comment]
剛好想做一個可以互相溝通的伺服器，這是個不錯的參考

**Comment by xkllkx** [comment]
謝謝分享，最近剛好有接觸相關的東西，覺得寫得很清楚。

**Comment by Yuhsin000** [comment]
最近上其他課也在講http耶，這個好酷喔，我之後也想試試看用websocker 產生一個網站

**Comment by ghost** [comment]
之後也想嘗試架構一個網站，有空來看看

**Comment by jason79461385** [comment]
網路概論的時候自己有做一個專題，是把專案推到heroku上運行的一個小型社群網站，如果想要用一個互動網站的話可以試試看heroku

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by etin2468** [comment]
感謝分享，我又多知道了一個網站的相關協議

**Comment by Hchejxnc** [comment]
感謝分享~~
